web/js/funblocks.js

```diff
@@ -290,10 +315,9 @@ function compile(src, silent) {
  * to get the visual presentation to match.
  */
 function updateUI() {
```

Comment:
So if I understand correctly, the updateUI in codeworld_shared is still used only by codeworld.js, and not by funblocks.js?  Or will calls to updateUI from within codeworld_shared.js still call the local one, but calls from here will call the override here?  That's quite confusing...